enhancedsteam.js

```diff
@@ -1511,6 +1514,26 @@ function add_wishlist_sorts() {
 			$("a.by_" + cookieVal).addClass("active");
 		}
 	}
+
+	function updateRankMode(sort_by) {
+		if (sort_by == "rank") {
+			runInPageContext(function() {
+				g_bRankMode = true;
+				Sortable.create($('wishlist_items'), {tag: 'div', scroll: window, onUpdate: UpdateWishlistOrdering, starteffect: function(e) {e.addClassName("inDrag")}, endeffect: function(e) {e.removeClassName("inDrag")}});
```

Comment:
This line is basically taken straight from Steam's JS. I don't think that's a problem since it's a pretty trivial line.